src/client/components/threatbar/threatbar.js

```diff
@@ -88,10 +88,10 @@ class Threatbar extends React.Component {
   }
 
   render() {
-    let threats = this.getThreatsForSelectedComponent();
-    let identifiedThreats = this.getIdentifiedThreatsForSelectedComponent();
-    let component = this.getSelectedComponent();
-    let componentName = getComponentName(component);
+    const threats = [...this.getThreatsForSelectedComponent()].reverse();
```

Comment:
I believe this is a point where I can learn something:
Why did you choose
`[...this.getThreatsForSelectedComponent()].reverse()`
over 
`this.getThreatsForSelectedComponent().reverse()`?

Doesn't `getThreatsForSelectedComponent` already return an array? Or is it for making the code more "type safe"?

Reply:
It improves robustness because `.reverse()` mutates the array in-place. So if the method just returned a reference to an array that is used elsewhere, the reversal would be seen there as well, which could break things. I think it's not a problem right now, but I wanted to make sure that future refactorings are safe. Unfortunately those built-in array methods often mutate the array. It's the same with `.sort()` (but not with `.slice()`, for example).